DESCRIPTION

```diff
@@ -2,7 +2,7 @@ Package: rvwgsoc
 Type: Package
 Title: R interface for Vowpal Wabbit
 Version: 0.4
-Date: 2018-07-23
+Date: 2018-07-31
 Author: Ivan Pavlov, Dirk Eddelbuettel, James J Balamuta
 Maintainer: Ivan Pavlov <pavlov.ivan095@gmail.com>
 Description: R interface for Vowpal Wabbit using Rcpp and libvw for GSoC 2018.
```

Comment:
One thing you see when you run `R CMD check --as-cran` (as one is asked to) is that it gets nit picky over this file. I see

```
   Possibly mis-spelled words in DESCRIPTION:
     GSoC (8:69)
     Rcpp (8:50)
     Vowpal (3:24, 8:30)
     libvw (8:59)
   
   The Title field should be in title case, current version then in title case:
   ‘R interface for Vowpal Wabbit’
   ‘R Interface for Vowpal Wabbit’
```

so you want to capitalize "Interface" in `Title:` and protect the terms "Vowpal Wabbit", "libvw", "Rcpp" and "GSoC 2018" with single quotes ie `'Vowpal Wabbit'` and so on.  It may make sense to expand terms too (ie "Google Summer of Code", "the 'Vowpal Wabbit' library 'libvw'", ...).